compiler/souschef/include/souschef/TensorFiller.h

```diff
@@ -85,7 +85,7 @@ class TensorFiller
     return false;
   }
 
-private:
+protected:
```

Comment:
Need for clear this fillers in TFliteImport when there is a subgraph changing

Reply:
how about provide a clear method to clear the map?